.gitignore

```diff
@@ -0,0 +1,398 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
is this the recommended gitignore of dotnet? 
i guess the command should be dotnet new gitignore ? 

check it out.